app/models/task/Task.scala

```diff
@@ -239,19 +239,36 @@ object TaskSQLDAO extends SQLDAO[TaskSQL, TasksRow, Tasks] {
     } yield parsed
   }
 
-  def findAllByPojectAndTaskTypeAndIds(projectOpt: Option[String], taskTypeOpt: Option[String], idsOpt: Option[List[String]])(implicit ctx: DBAccessContext): Fox[List[TaskSQL]] = {
+  def findAllByPojectAndTaskTypeAndIds(
```

Comment:
not sure if findAllByP**r**ojectAndTyskType**AndUser**AndIds is actually good, but maybe better^^